src/root-command.ts

```diff
@@ -56,7 +58,12 @@ export default abstract class extends Command {
   styledJSON(data: any) {
     const {flags} = this.parse()
     if (flags.silent) return
-    cli.styledJSON(data)
+    const base58EncodedHash = JSON.parse(JSON.stringify(data, (key: string, value: any): any => {
```

Comment:
If somebody has a better way instead of `JSON.parse(JSON.stringify`, please implement it :)